manifest/etcd.yaml

```diff
@@ -0,0 +1,148 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: etcd
+spec:
+  secretName: etcd-secret
```

Comment:
nit (global comment): try to avoid repeating the type of the resource in its name - e.g. use something like `etcd` or `etcd-certificate` instead of `etcd-secret`.